scripts/helper.js

```diff
@@ -637,13 +637,12 @@ HELPER.generate = (() => {
 			// iterate over all packages
 			if( allModules !== undefined && allModules.length > 0 ) {
 				for( let module of allModules ) {
-					const pkgPath = Path.normalize(`${ packagesPath }/${ module }/package.json`);
-					packageJson = require( pkgPath ); // read the package.json
+					const packageJson = require( Path.normalize( `${ packagesPath }/${ module }/package.json` ) );
```

Comment:
`packageJson` is being initialized at line 634...

Reply:
Yeah nice, missed that!